.pre-commit-config.yaml

```diff
@@ -34,6 +34,7 @@ repos:
   rev: 3.9.2
   hooks:
   - id: flake8
+    files: \.py$
```

Comment:
What does this do?

Reply:
It limits it to run on only python files.  We have the same on black and isort.